web/src/pages/index.page.tsx

```diff
@@ -33,7 +33,7 @@ const Page = () => {
 
       <TopicToolbar />
 
-      <Box flex={1} position="relative">
+      <Box height="100%" position="relative">
```

Comment:
np: I think it'd be good to use `const WorkspaceBox = styled(Box)` here to help establish the naming

Reply:
created a index.styles.tsx to hold custom component